gradle/libs.versions.toml

```diff
@@ -0,0 +1,24 @@
+[versions]
+agp = "7.4.2"
+dokka = "1.8.10"
+jdk = "19"
+kotlin = "1.8.10"
+leakcanary = "2.10"
+mavenPublish = "0.25.1"
+
+[plugins]
+agp-application = { id = "com.android.application", version.ref = "agp" }
+agp-library = { id = "com.android.library", version.ref = "agp" }
+dokka = { id = "org.jetbrains.dokka", version.ref = "dokka" }
+kotlin-android = { id = "org.jetbrains.kotlin.android", version.ref = "kotlin" }
+kotlin-jvm = { id = "org.jetbrains.kotlin.jvm", version.ref = "kotlin" }
+mavenPublish = { id = "com.vanniktech.maven.publish", version.ref = "mavenPublish" }
```

Comment:
I don't think this is needed anymore, I just use the built in maven publish plugin now as AGP correctly supports it.

Reply:
This is still useful if you want all the bells and whistles (dokka, snapshots, sonatype support, etc)